label_sync/labels.yaml

```diff
@@ -49,11 +50,22 @@ labels:
       - name: close/not-reproducible
     target: both
     addedBy: humans
+  - color: d455d0
+    description: Indicates an issue that is an support question that was opened incorrectly. The usage should be supported by adding triage findings as an issue comment.
```

Comment:
Nit - replace "an support question" to "a support question"? OR change the sentence to "Indicates an issue is a support question that was opened incorrectly."?